Models/Crime.cs

```diff
@@ -1,36 +1,24 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Threading.Tasks;
-// <auto-generated />
-//
-// To parse this JSON data, add NuGet 'Newtonsoft.Json' then do:
-//
-//    using QuickType;
-//
-//    var neighbourhoodAlert = NeighbourhoodAlert.FromJson(jsonString);
-namespace Neighbourhood_Alert
-
-
-
+﻿namespace NeighbourhoodCrime
 {
     using System;
     using System.Collections.Generic;
-
     using System.Globalization;
     using Newtonsoft.Json;
     using Newtonsoft.Json.Converters;
-
-    public partial class NeighbourhoodAlert
+    public partial class Crime
     {
+        private DateTimeOffset dateReported;
+        private string instanceid;
+        private string incidentNo;
+
         [JsonProperty("instanceid")]
-        public string Instanceid { get; set; }
+        public string Instanceid { get => instanceid; set => instanceid = value; }
```

Comment:
While encapsulation is a good idea, for our purposes, the old way is fine; using { get; set; } as a shortcut for encapsulation.